### ISSUE TITLE
A11y_Azure Tools Storage Explorer_AddResourceViaAzureActiveDirectory_AttachWithAzureAD _Zoom: Some of the content of the screen is not visible properly after zooming in the screen to 400%.

### ISSUE THREAD

**Issue by shreyashi227** [microsoft/AzureStorageExplorer#3477]
**GitHubTags:** #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.4.10;#Win32;#A11ySev1;#Benchmark;#Zoom;#FTP;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10

### **Prerequisite:**
1. Set display resolution to 1280x1024
2. Click on "Advanced scaling settings"
3. Enter 400% in the edit field "Enter a custom scaling size..."
4. Click on "sign out"

### **Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Add a resource via Azure Active Directory(Azure AD)" radio button and hit enter on Next button.
4. "Attach with Azure AD" screen gets opened.
5. Check if Some of the content of the screen is not visible properly after zooming in the screen to 400%.

### **Actual:**
Some of the content of the screen is not visible properly after zooming in the screen to 400%. Here list items under "Account" heading is not visible at 400% zoom.

**Similar issue is observed throughout the application.**

### **Expected:**
All the content of the screen which is available at 100% zoom should be visible properly after zooming in the screen to 400%.

### **User Impact:**
Low vision users will not be able to view some of the contents at 400% zoom.

### **Attachment:**
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shrkh_microsoft_com/Ef6q_F-VeaJAs17WeptxXKEBbWjCeTcYfv6Y-3TrfDQpFg?e=IQfbMJ
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shrkh_microsoft_com/ETilIxGSQMJKhqqwVR_JC8EB4GrWnOciGIDYy_AHUeV14A?e=7ZnzSb

**Comment by craxal** [comment]
Duplicate of #2618.

**Comment by fsteffi** [comment]
**Github tags:** #A11yMAS;

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
